pkg/lib/size/size.go

```diff
@@ -1,18 +0,0 @@
-package size
-
-import (
-	"github.com/pkg/errors"
-
-	"kubevirt.io/containerized-data-importer/pkg/importer"
-)
-
-// Size returns the size in bytes of the provided endpoint. If the endpoint was archived, compressed or
-// converted to qcow2 the original image size is returned.
-func Size(endpoint, accessKey, secKey string) (int64, error) {
```

Comment:
Not sad to see this stuff go.  But it is referenced in the README.  So should probably update that.

Reply:
Yes, readme needs updating.
@aglitke @davidvossel I don't recall who wanted the Size() func. It's fine to deprecate it but I think some notification would be a good idea. Eg let QE and kubevirt-ansible know of this api change.

Reply:
I will update the readme. I looked hard for anything that was actually referencing this code, and the only place I could find was the test for it. IIRC the original intent was for it to be used in some of the controllers to figure out the size for some operations (maybe report progress, who knows) but its a tiny wrapper around the data stream Size field, so there is not much use to it at this point.